dirty_cat/datasets/_ken_embeddings.py

```diff
@@ -1,57 +1,139 @@
 """
 Get the Wikipedia embeddings for feature augmentation.
 """
+from typing import Optional, Set
+
 import pandas as pd
 from sklearn.decomposition import PCA
 
 from dirty_cat.datasets import fetch_figshare
 
+# Required for ignoring lines too long in the docstrings
+# flake8: noqa: E501
 
-def get_ken_embeddings(
-    types=None,
-    exclude=None,
-    embedding_table_id="all_entities",
-    embedding_type_id=None,
-    pca_components=None,
-    suffix="",
+
+_correspondence_table_url = (
+    "https://raw.githubusercontent.com/dirty-cat/datasets"
+    "/master/data/ken_correspondence.csv"
+)
+
+
+def get_ken_table_aliases() -> Set[str]:
+    """Get the supported aliases of embedded entities tables.
+
+    These aliases can be using in subsequent functions (see section *See Also*).
+
+    Returns
+    -------
+    aliases: set of str
+        The aliases of the embedded entities tables.
+
+    See Also
+    --------
+    :func:`get_ken_types`
+        Helper function to search for entity types.
+    :func:`get_ken_embeddings`
+        Download Wikipedia embeddings by type.
+    """
+    correspondence = pd.read_csv(_correspondence_table_url)
+    return set(["all_entities"] + list(correspondence["table"].values))
+
+
+def get_ken_types(
+    search: str = None,
+    *,
+    embedding_table_id: str = "all_entities",
 ):
+    """Helper function to search for entity types.
+
+    Parameters
+    ----------
+    search : str, optional
+        Substring pattern that filters the types of entities.
```

Comment:
Maybe you can add a parameter exclude just like in the `get_ken_embeddings`. I found that this is very useful when you want to be a bit more precise: for instance search for "games" but exclude "card" games.

Reply:
And to make it more compatible, have the same names for these parameters. Maybe replace `types` with `search` in the get_ken_embeddings or rename the two to `search_types`?

Reply:
I've ended up leaving as is because to me, in `get_ken_types` we *search* for available types, while in `get_ken_embeddings`, we get the embeddings matching some *types*.
I did add the `exclude` parameter though, it's indeed relevant :)